rootfs-builder/alpine/Dockerfile.in

```diff
@@ -10,9 +10,24 @@ RUN apk update && apk add \
     make \
     bash \
     gcc \
-    musl-dev \
+    g++ wget tar \
+    cmake autoconf automake m4 coreutils binutils vim \
+    musl-dev musl libc-dev \
     linux-headers \
     apk-tools-static \
     libseccomp \
     libseccomp-dev \
     curl
+
+# RUN ln -svf /usr/bin/gcc /bin/musl-gcc; ln -svf /usr/bin/g++ /bin/musl-g++
```

Comment:
this `RUN` and the comment block below it look like they could be cleaned up a little - remove the blank lines at least, and move the words above the commented out `RUN` command if that is what they are referring to.

Reply:
We can delete the comment, since we cannot build rust agent on alpine